includes/class-factory.php

```diff
@@ -1,35 +1,42 @@
 <?php
 
-class WP_Notify_Factory {
+namespace WP\Notifications;
+
+use WP\Notifications\Messages\Message_Factory;
```

Comment:
It's a style thing really, but I personally always prefer to import namespaces rather than specific classes  - keeps things clearer, and also lets you name things a bit more concisely.

So rather than having 

```php
use WP\Notifications\Recipients\Recipient_Factory;
use WP\Notifications\Recipients\Recipient_Collection;

//...

/**
 * Recipient factory implementation to use.
 *
 * @var Recipient_Factory
 */
private $recipient_factory;

//...

$recipients = new Recipient_Collection();
```

You could rename the class and simplify imports to something like

```php

use WP\Notifications\Recipients;

//...

/**
 * Recipient factory implementation to use.
 *
 * @var Recipients\Factory
 */
private $recipient_factory;

//...

$recipients = new Recipients\Collection();
```

Reply:
I am really new to PHP namespaces, so I don't have any preferences. This works for me!

I'll make this change. Should classes in the global namespace be imported at the top of a file with:

```
use Exception;
```

or done inline?

```
\Exceptions
```

Reply:
I went with the use statement option for globals